.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[isort]
+multi_line_output = 2
```

Comment:
This comes from https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html and is the style we currently use in the repo.